tests/ApaiIO/Test/Operations/Types/LookupTest.php

```diff
@@ -28,6 +28,15 @@ public function testSetter()
         $lookup->setItemId('B1234');
     }
 
+    public function testSettersNegative()
+    {
+        $this->setExpectedException('\Exception');
+        $lookup = new Lookup();
+        $lookup->setItemIds(
+            [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

Comment:
The library still supports PHP 5.3 and the short array syntax doesn't work here.